examples/basic-infection/transmission_manager.rs

```diff
@@ -0,0 +1,62 @@
+use ixa::context::Context;
+use ixa::define_rng;
+use ixa::random::ContextRandomExt;
+
+use crate::people::ContextPeopleExt;
+use crate::people::InfectionStatus;
+use rand_distr::Exp;
+
+use crate::FOI;
+use crate::MAX_TIME;
+use crate::POPULATION;
+
+define_rng!(TransmissionRng);
+
+fn attempt_infection(context: &mut Context) {
+    let population_size: usize = context.get_population();
+    let person_to_infect: usize = context.sample_range(TransmissionRng, 0..population_size);
+
+    let person_status: InfectionStatus = context.get_person_status(person_to_infect);
+
+    if matches!(person_status, InfectionStatus::S) {
+        context.set_person_status(person_to_infect, InfectionStatus::I);
+        //context.schedule_recovery(person_to_infect);
+    }
+
+    #[allow(clippy::cast_precision_loss)]
+    let next_attempt_time = context.get_current_time()
+        + context.sample_distr(TransmissionRng, Exp::new(1.0 * FOI).unwrap()) / POPULATION as f64;
```

Comment:
Can you update the docs to reflect the change here? Also – can't you use `population_size` here?